src/ui/AnalyticsSuggestions/AnalyticsSuggestions.ts

```diff
@@ -58,8 +58,7 @@ export class AnalyticsSuggestions extends Component {
 
     /**
      * Specifies the title of the query suggestions group in the {@link Omnibox} component. This option is not available
-     * when using the default Lightning Friendly Theme (see
-     * [Lightning Friendly Theme](https://developers.coveo.com/x/Y4EAAg)).
+     * when using the default Lightning Friendly Theme.
```

Comment:
I have no idea what this even means hehe... I mean, I know it is C4SF-related, but I really have no clue what the "default Lightning Friendly Theme" is.

@samisayegh , @ThibodeauJF does that ring a bell to you?

Reply:
Not familiar with the theme per say. My guess is that the lightning theme is hiding, using CSS or js, the title that the search-ui displays when the option is specified.

So perhaps it's a warning to not expect anything to show when setting the option inside of lightning?